src/styles/catalog.scss

```
@@ -0,0 +1,15 @@
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: 4 * $width + (48px * 4);
```

```suggestion
  max-width: 4 * $width + (48px * 4);
```
48px also will be good to place to variable called `$gap`